src/tag/Type.h

```diff
@@ -46,6 +46,7 @@ enum TagType
 	TAG_NAME,
 	TAG_GENRE,
 	TAG_DATE,
+	TAG_ORIGINAL_DATE,
```

Comment:
Why would this need a preprocessor check? This is MPD's internal definition, not one imported from libmpdclient (which in turn mirrors MPD's internal definitions to use MPD's features).